<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SHREYAS-PC" failures="2" tests="3" name="Error Valications Test" time="102.114" errors="0" timestamp="2023-08-17T20:44:21 IST">
  <testcase classname="Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest" name="ProductValidation" time="14.638">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest.ProductValidation(ErrorValidationsTest.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- ProductValidation -->
  <testcase classname="Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest" name="ProductValidation" time="14.888">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest.ProductValidation(ErrorValidationsTest.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- ProductValidation -->
  <testcase classname="Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest" name="loginErrorValidation" time="1.29"/>
</testsuite> <!-- Error Valications Test -->
